ansible/configs/base-infra/post_software.yml

```diff
@@ -64,6 +64,8 @@
               targetport: "{{ hostvars[groups['bastions'][0]].bastion_ssh_port }}"
               subdomain: "{{ guid }}{{ subdomain_base_suffix }}"
               subdomain_internal: "{{ chomped_zone_internal_dns | default('') }}"
+              external_ip: "{{ hostvars[groups['bastions'][0]]['external_ip'] }}"
```

Comment:
Add | default('') so wont fail on other CIs

and we can add a when to the task to write the IP